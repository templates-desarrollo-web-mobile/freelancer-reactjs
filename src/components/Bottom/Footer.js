const Footer = () => (
  <div className="copyright py-4 text-center text-white">
    <div className="container">
      <small>Copyright &copy; Your Website 2021</small>
    </div>
  </div>
);

export default Footer;