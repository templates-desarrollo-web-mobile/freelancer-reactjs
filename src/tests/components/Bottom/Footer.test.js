import Footer from "../../../components/Bottom/Footer";
import { shallow } from 'enzyme';
describe('Pruebas en "Footer"', () => {
    
    const wrapper = shallow(<Footer/>);
    test('Debe de mostrar correctamente el componente', () => {
        
        expect( wrapper ).toMatchSnapshot();
    });

    test('Comprobar si texto del footer es el indicado', () => {
        const title = wrapper.find("small").text().trim();
        expect(title).toBe("Copyright Â© Your Website 2021");
    });

});