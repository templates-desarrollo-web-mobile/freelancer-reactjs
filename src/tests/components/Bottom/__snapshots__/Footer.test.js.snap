// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en "Footer" Debe de mostrar correctamente el componente 1`] = `
<div
  className="copyright py-4 text-center text-white"
>
  <div
    className="container"
  >
    <small>
      Copyright Â© Your Website 2021
    </small>
  </div>
</div>
`;
